suite-native/module-settings/src/tests/CurrencySelector.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
this test should be in folder `/src/components/tests/...` we are creating tests folders per folder

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/a54a081b813e4d5c0367770c18c3276d48792e06